code/modules/mob/living/carbon/human/human.dm

```diff
@@ -188,7 +188,7 @@
 		dat += "</td></tr>"
 
 	var/obj/item/bodypart/O = get_bodypart(BODY_ZONE_CHEST)
-	if((w_uniform == null && !(dna && dna.species.nojumpsuit) && !(O && O.status == BODYPART_ROBOTIC)) || (obscured & ITEM_SLOT_ICLOTHING))
+	if((w_uniform == null && !(dna?.species.nojumpsuit) && !(O?.status == BODYPART_ROBOTIC)) || (obscured & ITEM_SLOT_ICLOTHING))
```

Comment:
```suggestion
	if((w_uniform == null && !(dna?.species.nojumpsuit) && (!O || O.status != BODYPART_ROBOTIC)) || (obscured & ITEM_SLOT_ICLOTHING))
```
This check has further problems, but it's out of scope of this PR. However, the change may affect behavior.

Reply:
The `O && O.status == BODYPART_ROBOTIC` change part should be either reverted or the suggestion accepted.
We shouldn't rely on the fact that the value of `BODYPART_ROBOTIC` is currently not `null`